src/userdev/java/net/minecraftforge/gradle/userdev/UserDevPlugin.java

```diff
@@ -316,4 +341,25 @@ private NamedDomainObjectContainer<RenameJarInPlace> createReobfExtension(Projec
         project.getExtensions().add("reobf", reobfExtension);
         return reobfExtension;
     }
+
+    protected void configureJarJarTask(Project project) {
+        project.getConfigurations().create(JAR_JAR_DEFAULT_CONFIGURATION_NAME);
+
+        JavaPluginExtension extension = project.getExtensions().getByType(JavaPluginExtension.class);
+        project.getTasks().register(JAR_JAR_TASK_NAME, JarJar.class, jarJar -> {
+            jarJar.setGroup(JAR_JAR_GROUP);
+            jarJar.setDescription("Create a combined JAR of project and selected dependencies");
+            jarJar.getArchiveClassifier().convention("all");
+
+            jarJar.getManifest().inheritFrom(((Jar) project.getTasks().getByName("jar")).getManifest());
+            jarJar.from(extension.getSourceSets().getByName("main").getOutput());
+
+            jarJar.configuration(project.getConfigurations().getByName(JAR_JAR_DEFAULT_CONFIGURATION_NAME));
+            jarJar.exclude("META-INF/INDEX.LIST", "META-INF/*.SF", "META-INF/*.DSA", "META-INF/*.RSA", "module-info.class");
```

Comment:
Should this be configured for users automatically?

Reply:
That is indeed not needed anymore and a remnant from me testing this with shadow code.
Removing since it is not needed.